From c4354d356273697aee16e509a0db2e931a81e6e4 Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Tue, 8 Oct 2024 21:08:42 +0200
Subject: [PATCH] discover zip headers

---
 fairseq2n/src/fairseq2n/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fairseq2n/src/fairseq2n/CMakeLists.txt b/fairseq2n/src/fairseq2n/CMakeLists.txt
index dc99d3d5..068d9f63 100644
--- a/fairseq2n/src/fairseq2n/CMakeLists.txt
+++ b/fairseq2n/src/fairseq2n/CMakeLists.txt
@@ -103,6 +103,7 @@ target_include_directories(fairseq2n ${system}
 )
 
 target_include_directories(fairseq2n PRIVATE $ENV{PREFIX}/include)
+target_include_directories(fairseq2n PRIVATE $ENV{SRC_DIR}/fairseq2n/third-party)
 
 target_link_libraries(fairseq2n
     PRIVATE
-- 
2.39.5 (Apple Git-154)

