From 7aa050a51dac091727951866d81331e3bd8bc6de Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Fri, 11 Oct 2024 22:16:47 +0200
Subject: [PATCH 9/9] debug

---
 fairseq2n/CMakeLists.txt | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/fairseq2n/CMakeLists.txt b/fairseq2n/CMakeLists.txt
index 738817d7..4eb26faa 100644
--- a/fairseq2n/CMakeLists.txt
+++ b/fairseq2n/CMakeLists.txt
@@ -32,6 +32,8 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
 include(cmake/base.cmake)
 include(cmake/summary.cmake)
 
+message(STATUS " ------------------ GOT THIS FAR #0 ------------------")
+
 # Prefer pthread over other thread libraries.
 set(THREADS_PREFER_PTHREAD_FLAG TRUE)
 
@@ -101,6 +103,8 @@ option(FAIRSEQ2N_USE_CUDA
         OFF
 )
 
+message(STATUS " ------------------ GOT THIS FAR #1 ------------------")
+
 if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
     set(default_thread_lib tbb)
 else()
@@ -144,6 +148,9 @@ cmake_dependent_option(FAIRSEQ2N_PYTHON_DEVEL
         OFF
 )
 
+
+message(STATUS " ------------------ GOT THIS FAR #2 ------------------")
+
 # ------------------------------------------------------------
 # Sanitizers
 # ------------------------------------------------------------
@@ -168,7 +175,9 @@ find_package(Torch 1.13 REQUIRED)
 
 if(FAIRSEQ2N_BUILD_PYTHON_BINDINGS)
     set(Python_EXECUTABLE $ENV{PREFIX}/bin/python)
+    set(CMAKE_FIND_DEBUG_MODE TRUE)
     find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+    set(CMAKE_FIND_DEBUG_MODE FALSE)
     message(STATUS "Python_EXECUTABLE ${Python_EXECUTABLE}")
     if(Python_VERSION VERSION_LESS 3.8)
         message(FATAL_ERROR "fairseq2n requires CPython 3.8 or greater!")
@@ -194,7 +203,21 @@ find_package(fmt REQUIRED)
 
 # fairseq2n_add_kaldi_native_fbank()
 
+message(STATUS " ------------------ looking for kaldi-native-fbank ------------------ ")
 find_package(kaldi-native-fbank REQUIRED)
+if (kaldi-native-fbank_FOUND)
+    message(STATUS "kaldi-native-fbank was found!")
+
+    if (DEFINED kaldi-native-fbank_INCLUDE_DIR)
+        message(STATUS "Include dirs: ${kaldi-native-fbank_INCLUDE_DIR}")
+    endif()
+
+    if (DEFINED kaldi-native-fbank_LIB_PATH)
+        message(STATUS "lib path: ${kaldi-native-fbank_LIB_PATH}")
+    endif()
+else()
+    message(FATAL_ERROR "kaldi-native-fbank not found!")
+endif()
 
 fairseq2n_add_natsort()
 
-- 
2.39.5 (Apple Git-154)

